galsim/chromatic.py

```diff
@@ -1587,11 +1680,10 @@ def drawImage(self, bandpass, image=None, integrator='trapezoidal', **kwargs):
         add_to_image = kwargs.pop('add_to_image', False)
         # Use given add_to_image for the first one, then add_to_image=False for the rest.
         image = self.objlist[0].drawImage(
-                bandpass, image=image, add_to_image=add_to_image, **kwargs)
+            bandpass, image=image, add_to_image=add_to_image, **kwargs)
```

Comment:
In most cases, I prefer 8 spaces for a line continuation to better differentiate it from a new code block.  I think pep8 has the schizophrenic rule that line continuations should be 4 unless the next line is an intended code block, in which case it should be 8.  But having the general rule of 8 for all continuations seems cleaner.  In any case, I wouldn't want to proscribe 8-space indents such as this.